cmd/autoscaling.go

```diff
@@ -163,3 +185,12 @@ func printScaleConfig(cmdCtx *cmdctx.CmdContext, cfg *api.AutoscalingConfig) {
 		}
 	}
 }
+
+func printMachinesAutoscalingBanner() {
+	fmt.Printf(`
+Configuring autoscaling via 'flyctl autoscale' is supported only for apps running on Nomad platform.
+To enable autoscale for Apps V2 create the required number of machines in advance and allow fly.io
+to start and stop them on demand by setting auto_start_machine=true and auto_stop_machine=true
```

Comment:
could we link to the fresh-produce post instead of reexplaining this here? Ideally we should link to docs but they are in the making.